### ISSUE TITLE
Add a basic CI test to wrench ensure binary record / replay is working.

### ISSUE THREAD

**Issue by glennw** [servo/webrender#1253]

**Comment by glennw** [comment]
@kvark Do we want to do this anymore? I assume we'll deprecate / remove the binary record at some point in the future, so we probably don't need to do this?

**Comment by kvark** [comment]
@glennw I don't think we can remove this until the new capturing gets any sort of continuous recording. Let's keep this open with low priority and "help needed"? It doesn't appear to involve a ton of work.
